### PR TITLE
Add basic latency metrics to scheduler.

### PR DESCRIPTION
**Pull request by fgrzadkowski**
This change is needed to properly understand low scheduler throughput (#3954)

### CODE REVIEW

**Comment by vmarmol** [comment]
LGTM

**Comment by vmarmol** [comment]
You can't aggregate summaries over more than one instance, but since we only have one scheduler it should be fine and easier for now.

File: plugin/cmd/kube-scheduler/app/server.go [hunk 1]
```diff
@@ -80,11 +81,11 @@ func (s *SchedulerServer) Run(_ []string) error {
 
 	go func() {
 		if s.EnableProfiling {
-			mux := http.NewServeMux()
-			mux.HandleFunc("/debug/pprof/", pprof.Index)
-			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
-			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+			http.HandleFunc("/debug/pprof/", pprof.Index)
+			http.HandleFunc("/debug/pprof/profile", pprof.Profile)
+			http.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
```
Comment (timothysc) [inline plugin/cmd/kube-scheduler/app/server.go]: I believe this recent change has been causing a startup crash we've seen.

Comment (fgrzadkowski) [inline plugin/cmd/kube-scheduler/app/server.go]: Can you say something more? What crash? What message?

Comment (wojtek-t) [inline plugin/cmd/kube-scheduler/app/server.go]: I've just faced the same. The issue is:

    panic: http: multiple registrations for /debug/pprof/
goroutine 11 [running]:
net/http.(*ServeMux).Handle(0xc20800a8a0, 0x9e9490, 0xd, 0x7f7d12f190a0, 0xb0e7c0)
        /usr/src/go/src/net/http/server.go:1557 +0x244
net/http.(*ServeMux).HandleFunc(0xc20800a8a0, 0x9e9490, 0xd, 0xb0e7c0)
        /usr/src/go/src/net/http/server.go:1585 +0x6d
net/http.HandleFunc(0x9e9490, 0xd, 0xb0e7c0)
        /usr/src/go/src/net/http/server.go:1597 +0x48
github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app.func·001()
        /go/src/github.com/GoogleCloudPlatform/kubernetes/_output/dockerized/go/src/github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app/server.go:84 +0x58
created by github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app.(*SchedulerServer).Run
        /go/src/github.com/GoogleCloudPlatform/kubernetes/_output/dockerized/go/src/github.com/GoogleCloudPlatform/kubernetes/plugin/cmd/kube-scheduler/app/server.go:90 +0x1c7

Comment (wojtek-t) [inline plugin/cmd/kube-scheduler/app/server.go]: I've filed an issue for it #6621

**Pull request merged**